Marlin/src/inc/SanityCheck.h

```diff
@@ -2143,7 +2143,7 @@ static_assert(Y_MAX_LENGTH >= Y_BED_SIZE, "Movement bounds (Y_MIN_POS, Y_MAX_POS
   #error "STEALTHCHOP requires TMC2130, TMC2160, TMC2208, TMC2209, or TMC5160 stepper drivers."
 #endif
 
-#define IN_CHAIN(A) (A##_CHAIN_POS > 0)
+#define IN_CHAIN(A) ((A##_CHAIN_POS > 0) && !HAS_L64XX)
```

Comment:
According to this change the presence of any L64XX stepper drivers makes `IN_CHAIN` always `false`. Is this because the L64XX and TMC drivers can only be combined in the same bus with regular non-chaining?

Reply:
TMC and L64xx can't be used in the same chain.  All TMC SPI chips (except for one device) have a 40 bit delay when repeating MOSI on MISO while the L64xx has an 8 bit delay.

L64xx doesn't use the IN_CHAIN(A) macro, only the TMC devices do.  

Sanity check will fail an L64xx axis without the added term.